### PR TITLE
Add netinfo/OWNERS

### PR DESCRIPTION
**Pull request by zcorpan**
cc @ferjm

### CODE REVIEW

**Review by jkarlin — APPROVED** [review]
lgtm

File: netinfo/OWNERS [hunk 1]
```diff
@@ -0,0 +1,4 @@
+@jkarlin
+@igrigorik
+@marcoscaceres
+@ferjm
```
Comment (igrigorik) [inline netinfo/OWNERS]: I'd also like to nominate @tarunban. :)

**Review by marcoscaceres — APPROVED** [review]
Ok from me

File: netinfo/OWNERS (hunk 1 above)
Comment (marcoscaceres) [reply netinfo/OWNERS]: Done

**Review by igrigorik — APPROVED** [review]
lgtm!

File: netinfo/OWNERS (hunk 1 above)
Comment (igrigorik) [reply netinfo/OWNERS]: ty!

**Comment by tarunban** [comment]
thank you!

**Review by ferjm — CHANGES_REQUESTED** [review]
I am not following this spec anymore, so please remove my name from the list of owners. Thanks!

**Comment by zcorpan** [comment]
Done

**Pull request merged**